src/wayland_egl/pxWindowNative.cpp

```diff
@@ -591,7 +591,7 @@ void pxWindowNative::runEventLoop()
     int pollResult = 0;
 #endif //PXCORE_WL_DISPLAY_READ_EVENTS
     double maxSleepTime = (1000 / framerate) * 1000;
-    rtLogInfo("max sleep time in microseconds: %f", maxSleepTime);
+    rtLogDebug("max sleep time in microseconds: %f", maxSleepTime);

```

Comment:
Please leave this logging statement as info.  It's important for it to be logged by default at the info level, and it's only logged once at startup.